src/wp-admin/includes/class-plugin-upgrader.php

```diff
@@ -444,7 +444,7 @@ public function bulk_upgrade( $plugins, $args = array() ) {
 			unset( $past_failure_emails[ $plugin ] );
 		}
 
-		update_option( 'auto_plugin_theme_update_emails', $past_failure_emails );
+		update_option( 'auto_plugin_theme_update_emails', $past_failure_emails, false );
```

Comment:
This still needs to be reverted, see https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731865969.
```suggestion
		update_option( 'auto_plugin_theme_update_emails', $past_failure_emails );
```